lib/kitchen/directions/bake_chapter_references/v1.rb

```diff
@@ -4,28 +4,51 @@ module Kitchen::Directions::BakeChapterReferences
   class V1
     renderable
 
-    def bake(chapter:, metadata_source:, uuid_prefix: '.', klass: 'references')
+    def bake(
+      chapter:,
+      metadata_source:,
+      uuid_prefix: '.',
+      klass: 'references',
+      from_introduction: false
+    )
       @metadata = metadata_source.children_to_keep.copy
       @klass = klass
       @title = I18n.t(:references)
       @uuid_prefix = uuid_prefix
 
-      chapter.references.search('h3').trash
+      chapter.references.search('h3').each(&:trash)
 
       chapter.non_introduction_pages.each do |page|
         references = page.references
         next if references.none?
 
-        references.search('h3').trash
         title = Kitchen::Directions::EocSectionTitleLinkSnippet.v1(page: page)
 
         references.each do |reference|
           reference.prepend(child: title)
         end
       end
 
-      @content = chapter.pages.references.cut.paste
+      if from_introduction == true
```

Comment:
per above, this block of code is no longer required.  But for the future, this line can instead simply be

```ruby
if from_introduction
```

you don't have to compare it to `true` because `from_introduction` is already a boolean.